src/app/shared/components/pricing-plans/pricing-plans.component.html

```diff
@@ -206,7 +206,7 @@ <h3 class="heading mt-4" [translate]="'page_pricing.features'">Features</h3>
                       >/m
                     </div>
                     <!--<div>${{plan.annually_price/12}}/m billed as <br>${{plan.annually_price}}/y</div>-->
-                    <div class="mt-3">
+                    <div class="mt-3" *ngIf="!isLifetimePrime">
```

Comment:
@superdev0714 
You can see in the code commented above this line how it supports the chat discussion and this is not indicating the current billing period but the price, because it is different when billed monthly.